### PR TITLE
Persist Weapons

### PR DESCRIPTION
**Pull request by gir489returns**
Updated GetVehicleInfoForClanLogo.
Made Ammunation into a CollapsingHeader.

### CODE REVIEW

File: src/services/persist_weapons/persist_weapons.h
Comment (Yimura) [inline src/services/persist_weapons/persist_weapons.h:1]: Use explicit CPP header `.hpp`.

File: src/services/persist_weapons/weaponloadout_json.h
Comment (Yimura) [inline src/services/persist_weapons/weaponloadout_json.h:1]: Use explicit CPP header `.hpp`.

**Review by Yimura — CHANGES_REQUESTED** [review]

**Comment by DayibBaba** [comment]
I wonder if this could be done with other players. Idk whether we can gather data on the weapons of other players, but im certain we can give them with a network event.

**Comment by gir489returns** [comment]
> I wonder if this could be done with other players. Idk whether we can gather data on the weapons of other players, but im certain we can give them with a network event.

You cannot give them components or set the tint. You can only give them weapons and the ammo for it. I tried that before when developing BBv2.

**Review by Yimura — DISMISSED** [review]

**Review by Yimura — APPROVED** [review]

**Pull request merged**